db/migrate/20131002215400_clean_openvas_settings.rb

```diff
@@ -1,4 +1,4 @@
-class CleanOpenVASSettings < ActiveRecord::Migration[5.1]
+class CleanLegacyConfig < ActiveRecord::Migration[5.1]
```

Comment:
why not change the file name to `db/migrate/20131002215400_clean_legacy_config.rb` if we're already changing the class name?

Reply:
Mainly because it doesn't describe the problem enough. With this trickery the filename remains descriptive, and Rails find the right class through the trickery in L14-15